components/list/bullet/src/index.scss

```diff
@@ -30,12 +30,18 @@ $sz-list-bullet-illustration-l: 72px !default;
       #{$prefix}-illustration {
         height: $sz-list-bullet-illustration-l;
         margin-bottom: $m-xl;
-        margin-right: 0;
```

Comment:
We need this specification to work properly on desktop-responsive

Reply:
fixed. well seen mate.